lib/util.js

```diff
@@ -64,6 +64,7 @@ const { debuglog } = require('internal/util/debuglog');
 const {
   validateFunction,
   validateNumber,
+  validateString
```

Comment:
```suggestion
  validateString,
```

Reply:
It looks like this has been resolved but not updated.